### ISSUE TITLE
Bug breaks all future sorting chests

### ISSUE THREAD

**Issue by Kreidos** [Dynious/RefinedRelocation#397]
I'm not sure which action did it, but either Picking up a sorting chest with a Mekanism cardboard box and placing it or placing some witchery items in a sorting chest seems to break all future sorting chests forever. Restarting server does not fix the issue. Previously placed chests are not affected, but any sorting chests placed in the future will not sort correctly and will display unpredictable behavior. This issue carries into all other worlds and saves. 

As chest functionality is broken on both my server and client, I am unable to try and ascertain which precise action caused the issue. Creating a new world does not restore normal function.

**Comment by blay09** [comment]
There's been reports on chests completely stopping to work, but I was never able to consistently reproduce it. The fact that it's broken in all other worlds is either a coincidence or based on using similar setups though - there is nothing that's persistent between worlds apart from the config file, which is not being modified from ingame. However, if you are running on the latest version and currently manage to reproduce the bug on all worlds created, would it be possible for you to create a small world with a setup that reproduces the issue and upload it somewhere, so I can have a look at it in my development environment?

#376

**Comment by Kreidos** [comment]
Thank you for your quick reply, and I apologize for my late one. I have a world which I've made which demonstrates the problem. I have the Resonant Rise 3 modpack, but I am running the latest Refined Relocation version and have only used RR and vanilla items.

Steps I've performed in this world: 
- Place 5 sorting iron chests in a row.
- Set the rightmost chest to whitelist All Ingots
- Set the chest second from the left to blacklist all ingots.

Tests:
Action performed: Place stacks of ingots in the leftmost chest.
Expected effect: Ingots move to the rightmost chest.
Observed effect: Ingots remain in the leftmost chest.

Action performed: Place stacks of ingots in the second chest (The one with ingots blacklisted)
Expected effect: Ingots move to the rightmost chest.
Observed effect: Ingots move to the leftmost chest.

World file is here: [NewWorld.zip](http://lancast.us.to/minecraft/NewWorld.zip)
